tests/test_object_detection.py

```diff
@@ -0,0 +1,37 @@
+from cleanlab.object_detection.rank import (
+    get_label_quality_scores,
+    issues_from_scores,
+)
+
+import numpy as np
+
+import warnings
+import pickle
```

Comment:
need to fetch the files from S3

Reply:
you can also put some smaller assets in this repo:
https://github.com/cleanlab/assets